KubeArmor/enforcer/appArmorEnforcer.go

```diff
@@ -340,28 +339,27 @@ func (ae *AppArmorEnforcer) CreateAppArmorHostProfile() error {
 		return nil
 	}
 
-	apparmorHostDefault := "## == Managed by KubeArmor == ##\n" +
-		"\n" +
-		"#include <tunables/global>\n" +
-		"\n" +
-		"profile kubearmor.host /{usr/,}bin/*sh flags=(attach_disconnected,mediate_deleted) {\n" +
-		"  ## == PRE START == ##\n" +
-		"  #include <abstractions/base>\n" +
-		"  mount,\n" +
-		"  umount,\n" +
-		"  signal,\n" +
-		"  unix,\n" +
-		"  ptrace,\n" +
-		"\n" +
-		"  file,\n" +
-		"  network,\n" +
-		"  capability,\n" +
-		"  ## == PRE END == ##\n" +
-		"\n" +
-		"  ## == POLICY START == ##\n" +
-		"  ## == POLICY END == ##\n" +
-		"}\n"
+	apparmorHostDefault := `## == Managed by KubeArmor == ##
+#include <tunables/global>
+
+profile kubearmor.host /{usr/,}bin/*sh flags=(attach_disconnected,mediate_deleted) {
+## == PRE START == ##
+#include <abstractions/base>
+mount,
+umount,
+signal,
+unix,
+ptrace,
 
+file,
+network,
+capability,
+## == PRE END == ##
+
+## == POLICY START == ##
+## == POLICY END == ##
+}
+	`
```

Comment:
@Revolyssup Can we handle this? as @kranurag7 pointed out?

```suggestion
}
`
```

Reply:
@daemon1024 Sorry for the delay. Yeah I will make the changes today. Thanks for the reminder.